modules/ppcp-save-payment-methods/src/Service/PaymentMethodTokensChecker.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Service for checking payment method tokens.
+ *
+ * @package WooCommerce\PayPalCommerce\SavePaymentMethods\Helper
+ */
+
+declare(strict_types=1);
+
+namespace WooCommerce\PayPalCommerce\SavePaymentMethods\Service;
+
+use WooCommerce\PayPalCommerce\ApiClient\Endpoint\PaymentTokensEndpoint;
+
+/**
+ * Class PaymentMethodTokensChecker.
+ */
+class PaymentMethodTokensChecker {
+
+	/**
+	 * Payment method tokens endpoint.
+	 *
+	 * @var PaymentTokensEndpoint
+	 */
+	private PaymentTokensEndpoint $payment_method_tokens_endpoint;
+
+	/**
+	 * PaymentMethodTokensChecker constructor.
+	 *
+	 * @param PaymentTokensEndpoint $payment_method_tokens_endpoint Payment method tokens endpoint.
+	 */
```

Comment:
After @AlexP11223 did [this](https://github.com/woocommerce/woocommerce-paypal-payments/pull/3474/files#diff-05ae9cddcaec1e845771a7db224961439f83ef5939ec67d3a48744cb34d7e58b) change we can avoid adding unnecessary constructor comments